scripts/travisci/check_flake8.sh

```diff
@@ -0,0 +1,19 @@
+exit_status=0
+# Absolute verification
+# W191 - Indentation contains tabs
+# E101 - Indentation contains mixed spaces and tabs
+find -iname "*.py" | flake8 --select=E101,W191
+if [ $? -ne 0 ]; then
+  exit_status=1
+fi
+
+# Incremental verification
+# E501 - Line too long
+if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
+  git diff ${TRAVIS_COMMIT_RANGE} | flake8 --diff --select=E501
```

Comment:
what about the `else` clause? You only covered pull request jobs (there are also branch jobs), which unfortunately need a slightly different way of calculating changed files).

See https://github.com/ryanjulian/rllab/commit/4694327875be9557fe1a7f805f23112bf223541a

Reply:
Done